low-level/json_msgs/messages/sensors/thread_monitor_msg.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import json
+import socket
+import time
+
+from json_msgs.messages.sensors.base_sensors_msg import BaseSensorMsg
+from framework.utils.mon_utils import MonUtils
+
+
+class ThreadMonitorMsg(BaseSensorMsg):
+
+    """The JSON message transmitted for module failure or recovery."""
+
+    MESSAGE_VERSION = "1.0.0"
+    RESOURCE_TYPE = "node:sspl:module"
+
+    def __init__(self, info, username="SSPL-LL", signature="N/A", expires=-1):
```

Comment:
Is it linux sspl user name? should be sspl-ll?

Reply:
Yes, like used in other sensor and actuator messages.